runtime/gc_realtime/RealtimeMarkingScheme.hpp

```diff
@@ -51,8 +50,7 @@ class MM_RealtimeMarkingScheme : public MM_SegregatedMarkingScheme
 private:
 	MM_RealtimeGC *_realtimeGC;  /**< The GC that this markingScheme is associated*/
 	MM_Scheduler *_scheduler;
-	J9JavaVM *_javaVM;  /**< The current JVM*/
-	MM_GCExtensions *_gcExtensions;
+	MM_GCExtensionsBase *_gcExtensions;
```

Comment:
looks like this variable is not necessary. The hierarchy is:
`MM_RealtimeMarkingScheme` <- `MM_SegregatedMarkingScheme` <- `MM_MarkingScheme`
and `MM_MarkingScheme` has `MM_GCExtensionsBase *_extensions;` already. It can be used instead